generated/nidigital/README.rst

```diff
@@ -0,0 +1,185 @@
+Overall Status
```

Comment:
Other files are shown in diff as: "File renamed without changes."
Except this one. What gives?

Reply:
`README.rst`'s are new to generated so they really are an add.

We need them in git though so that building the wheel will work without having to generate any files.

Reply:
ah ok